.svgrrc.yml

```diff
@@ -0,0 +1,33 @@
+svgoConfig:
+  multipass: true
+  plugins:
+    # Plugins included in preset-default
+    - removeDoctype
+    - removeXMLProcInst
+    - removeComments
+    - removeMetadata
+    - removeEditorsNSData
+    - cleanupAttrs
+      # minification may cause IDs to clash, so we disable it
+    - name: cleanupIds
+      params:
+        minify: false
+    - removeUselessDefs
+    - cleanupNumericValues
+    - removeUnknownsAndDefaults
+    - removeNonInheritableGroupAttrs
+    - removeUselessStrokeAndFill
+    - removeEmptyText
+    - collapseGroups
```

Comment:
i would not enable collapseGroups, as we use 'useless' groups for animation frequently

Reply:
wait nvm that's a different plugin